docs/user/app_getting_started.md

```diff
@@ -120,25 +123,17 @@ The **Custom Labels** tab provides options to configure a custom label range usi
 
 With an X Size of 10:
 
-```json
-[{"start": "02A", "end": "02J", "step": 1, "increment_letter": true, "label_type": "numalpha"}]
-```
+![numalpha_x_10](../images/numalpha_x_10.png)
```

Comment:
If feasible, consider giving more descriptive labels to these images for the benefit of users using screen readers and the like.